src/Core/Components/List/FluentCombobox.razor

```diff
@@ -10,7 +10,7 @@
 }
 
 <CascadingValue Value=@_internalListContext Name="ListContext" TValue="InternalListContext<TOption>" IsFixed=true>
-    <FluentInputLabel ForId="@Id" Label="@Label" AriaLabel="@AriaLabel" ChildContent="@LabelTemplate" />
+    <FluentInputLabel ForId="@Id" Label="@Label" AriaLabel="@AriaLabel" ChildContent="@LabelTemplate" Required=@Required />
```

Comment:
To be coherent with other attributes, could you add double-quotes: `Required="@Required"`?